### ISSUE TITLE
Fs22 Version

### ISSUE THREAD

**Issue by ornellino** [StjerneIdioten/AdjustableMirrors#42]
Hi, have you any plans to convert the mod for FS22?
Tanks

**Comment by StjerneIdioten** [comment]
> Hi, have you any plans to convert the mod for FS22? Tanks

Yes I do! Currently there is a version [here](https://github.com/StjerneIdioten/AdjustableMirrors/releases/tag/v2.0.0.0-dev) if you just want to play with it in single-player. I unfortunately don't have time right now to update the mod for multiplayer in FS22 and I probably won't put it on the modhub before that is added.

**Comment by ornellino** [comment]
Tanks! I checked the last commit date but not the releases!

**Comment by SBlack08** [comment]
hey there, is there anything new? can we still expect a release? thanks

**Comment by SbSh-DNA-ModDing** [comment]
@SBlack08  
https://github.com/StjerneIdioten/AdjustableMirrors/issues/42#issuecomment-1113856792

**Comment by StjerneIdioten** [comment]
> hey there, is there anything new? can we still expect a release? thanks

It is actually more or less done. I've rewritten a bunch of code and optimised some of the networking a bit. I need to test it on a dedicated multiplayer server first before I throw it on modhub though.

**Comment by SbSh-DNA-ModDing** [comment]
If you need a temp dedi for testing, let me know

**Comment by StjerneIdioten** [comment]
> If you need a temp dedi for testing, let me know

Thanks for the offer.

My friend is letting me borrow his dedi, I just don't have the time this weekend test it out. But hopefully there will be time sometime next week :-)

**Comment by thebikwirm** [comment]
any luck on this?
would love to seen a working mirror mod for fs22

**Comment by StjerneIdioten** [comment]
> any luck on this? would love to seen a working mirror mod for fs22

Yes, I have finished the work and sent it to the Modhub. I can see that the current status is that they have tested it once and are testing it again 😃 

In the meantime I've made a release [here](https://github.com/StjerneIdioten/AdjustableMirrors/releases/tag/FS22%2Fv3.0.0.0), if you just want to get going with it right away. It might be that Giants finds some stuff that needs to be fixed, but in the testing I've those this works fine in both singleplayer and multiplayer (co-op and dedicated).
